### ISSUE TITLE
SOS configuration needs refresh

### ISSUE THREAD

**Issue by morrone** [ovis-hpc/ovis#114]
In v4, the ldms/configure.ac still has the following:

```
OPTION_DEFAULT_DISABLE([sos], [ENABLE_SOS])
OPTION_WITH_OR_BUILD([sos],[../sos],[sos/src sos/include ods/src ods/include])
```

The "../sos" location is left over from the days before sos was moved into its own repository.  It should no longer be the default location.

Also, we should now remove the "--enable/disable-sos" option altogether.  Now that sos is an external package from ovis/ldms, there should only be a "--with/without-sos" option.

If "--with-sos" is specified without a provided path, the system should test for sos installed in the standard system location.  An alternate location can be specified with "--with-sos=/some/other/place".

**Comment by baallan** [comment]
agreed. I propose the fix is deferred until after #112 is merged.
